trpcultivate_genotypes/src/GenotypesLoader/GenotypesLoaderPluginBase.php

```diff
@@ -42,33 +43,41 @@ abstract class GenotypesLoaderPluginBase extends PluginBase implements Genotypes
    * The cvterm_id of the subtype of marker of the genotypes being inserted
    * For example, if the marker type is genetic_marker, the subtype can be one of "Exome Capture", "GBS", "KASPar", etc.
    * This must already exist.
-   * 
+   *
    * @var integer
    */
   protected $marker_subtype_id;
 
   /**
    * The type of input file containing the genotypes
    * Currently, this must be one of: vcf, matrix, legacy
-   * 
+   *
    * @var string
    */
   protected $file_type;
 
   /**
    * The filepath of the input file containing the genotypes
-   * 
+   *
    * @var string
    */
   protected $input_file;
 
   /**
    * The filepath of the tab-delimited file specifying each sample name in the genotypes file
-   * 
+   *
    * @var string
    */
   protected $sample_file;
 
+  /**
+   * An array of the samples provided in the samples file, associated with their stock IDs
```

Comment:
It might be nice to make it extra clear that the sample source name is the name in the genotypes file to be uploaded.

Reply:
This led me down a bit of a rabbit hole 😝 Apparently I was being inconsistent with the samples array (in terms of name AND contents). This is partially due to saving the sample name in Chado in the previous version of the genotypes loader- but, I confirmed this information never gets used! 

We've decided to stick to a simple association between the sample's DNA source name in the genotypes file and its stock ID in Chado. In commit https://github.com/TripalCultivate/TripalCultivate-Genetics/pull/26/commits/c704c3db0bb924d0ef5844d7039240121685ee67 I updated the code (including tests) to be consistent with this.